src/components/ColumnsEditor.tsx

```diff
@@ -16,7 +17,7 @@ export function CommandDisplayer({filledCommands, style}) {
     );
 }
 
-export function PythonDisplayer({filledCommands, style, getPyRequester}) {
+export function PythonDisplayer({filledCommands, style, getPyRequester}) {//
```

Comment:
Python displayer is a decent name.

getPyRequester is nasty.  that's the only way I could figure out how to integrate the backbone based ipywidgets with my react components.  Would love a better pattern there, open to a better name.